packages/ia-components/components/theatres/with-youtube-spotify/utils/get-track-list-by-source.js

```diff
@@ -9,7 +9,13 @@ const formatTime = (lengthInSeconds) => {
   const seconds = ((lengthInMS % 60000) / 1000).toFixed(0); // becomes a string, thanks JS math
   const extraMinute = seconds === '60';
   const secondsDisplay = extraMinute ? '00' : (seconds < 10 ? '0' : '') + seconds;
-  const minutesDisplay = extraMinute ? minutes + 1 : minutes;
+  let minutesDisplay = minutes;
+  if (!minutesDisplay) {
+    minutesDisplay = '00';
+  }
+  if (extraMinute) {
+    minutesDisplay = minutes + 1;
```

Comment:
I know this has already been merged and that it takes advantage of javascript's dynamic typing, but I think what @bfalling was pointing out is that `minutesDisplay`'s type changes several times so it becomes difficult to understand where it ends up in the end. It starts off as a number, then becomes a string, then becomes a number, then gets written out as a string. When you add in mathematic operations in there, there's a greater likeliness of unintended output.

For consistency, since `minutesDisplay` was intended to be a string, I think doing something like this would keep that obvious:
```
let minutesDisplay = minutes ? `${minutes}` : '00';
if (extraMinute) {
  minutesDisplay = `${minutes + 1}`;
}
```
It keeps `minutes` as a number and `minutesDisplay` as a string and when you need `minutes` to become a string, you wrap it in the `${minutes}`.

Reply:
hi, thanks for checking in. 
Q: so, the return will always be a string value, didn't think to reason about value types during processing until the end..
I see what you're saying tho re: satisfy keeping `minutesDisplay` as string through the calculation, sure, we can do that.  here's the [PR](https://github.com/internetarchive/iaux/pull/606) 🙂